docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -0,0 +1,87 @@
+
+This article provides details of the API ML OIDC authentication functionality and related configuration needed to achieve it.
+
+- [OIDC Usage](#oidc-usage)
+- [Authentication Workflow](#authentication-flow)
+- [Configuration](#api-ml-oidc-configuration)
+    * [Prerequisites](#prerequisites)
+    * [OIDC provider configuration](#oidc-provider-configuration)
+    * [API ML GW configuration](#api-ml-gw-configuration)
+    * [SAF/ESM configuration](#esm-configuration)
+- [Troubleshooting](#troubleshooting)
+
+## OIDC Usage
+OIDC authentication is used together with the z/OS [Identity Propagation](https://www.redbooks.ibm.com/redbooks/pdfs/sg247850.pdf) feature to implement the AI ML Identity Federation.
+[OIDC](https://openid.net/specs/openid-connect-core-1_0.html) protocol adds an authentication layer on top of the [OAuth2](https://www.rfc-editor.org/rfc/rfc6749) Authorization protocol.
+It is used by APIML client applications to verify user's identity with a distributed OIDC provider trusted by the mainframe security.
+After successful login the OIDC provider grants the client an Access Token (along with Identity Token, eventually also a Refresh Token if requested and appropriate).
+The Access Token issued by a trusted OIDC provider is now accepted by APIML.
+is the foundation of API ML multi-domain Identity Federation.
+The JWT Access Token obtained from the OIDC/IDP can be then used to authenticate the user (and authorize the client application) and allow access to mainframe resources through the API ML GW.
+
+## Authentication Flow
+The following diagram describes the interactions between the participants of the OIDC based API ML authentication process.
+
+<img src={require("../../images/api-mediation/apiml-oidc-auth-seq.png").default} alt="APIML OIDC Workflow" width="700"/>
+
+* When the User wants to access mainframe resources or services using the client application without a valid authentication / access token, 
+the client redirects the User agent to the login end-point of the distributed OIDC provider. 
+* The user is asked to provide their valid credentials (authentication factors).
+* After successful validation of all authentication factors, the OIDC provider grants the Client an access token.
+* The client then can request from API ML GW the needed mainframe resources presenting the access token in the request. 
+* The GW validates the access token at the OIDC provider's /introspection end-point and in case of success, caches the outcome for a short time. 
+* In subsequent calls with the same token, the GW reuses the cached validation outcome. This helps to save round trips to the OIDC /introspection end-point 
+for a short interval of time, when the client needs to access multiple resources in a row to complete a unit of work. The caching interval is configurable with a sensible default. 
```

Comment:
What is a "sensible" default? Can this default be more specific?

Reply:
Changed to a concrete value of the default plus explanation. See next version.